spec/spec_helper.rb

```diff
@@ -1,4 +1,5 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+require 'byebug'
```

Comment:
i'd rather not require byebug (we can keep it as a dev dependency) as it interfere's with some ability to debug if the dev is using rubymine